Rakefile.d/cucumber.rake

```diff
@@ -17,14 +17,16 @@ namespace :cucumber do
     sh "BUNDLE_GEMFILE=#{current_gemfile} bundle show cucumber > /tmp/cukes_version"
     sh "BUNDLE_GEMFILE=#{current_gemfile} bundle show sqlite3 > /tmp/sqlite_version"
     cukes_version=File.read('/tmp/cukes_version').split('-').last.chomp
-    sqlite_version=File.read('/tmp/sqlite_version').split('-').last.chomp
+    # Example 'sqlite_version' output:
+    #   /Users/username/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/sqlite3-1.6.8-arm64-darwin
+    sqlite_version=File.basename(File.read('/tmp/sqlite_version')).split('-')[1].chomp
     puts "Versions: Cucumber #{cukes_version}, SQLite #{sqlite_version}."
     Bundler.with_clean_env do
       gemfile = "cucumber_test_app/Gemfile"
       rm_rf "cucumber_test_app"
       sh "BUNDLE_GEMFILE=#{current_gemfile} bundle exec rails new cucumber_test_app --skip-spring --skip-javascript --skip-sprockets --skip-bootsnap"
       sh "echo 'gem \"cucumber\", \"~> #{cukes_version}\", :require => false' >> #{gemfile}"
-      sh "sed -i 's/gem .sqlite3./gem \"sqlite3\", \"~> #{sqlite_version}\"/' #{gemfile}"
+      sh "sed -i'' -e 's/gem .sqlite3./gem \"sqlite3\", \"~> #{sqlite_version}\"/' #{gemfile}"
```

Comment:
`sed` has slightly different implementation on macos and linux.
This command should work on both macos and linux.

Without this we will get an error:

```
sed -i 's/gem .sqlite3./gem "sqlite3", "~> darwin"/' cucumber_test_app/Gemfile
sed: 1: "cucumber_test_app/Gemfile": command c expects \ followed by text
rake aborted!
```

Reply:
oh, Mac, what an annoyance... I used `gsed` on my work laptop.... Happy to learn there is a common pattern though